ruma-common/src/push.rs

```diff
@@ -206,6 +211,159 @@ impl Serialize for Action {
     }
 }
 
+/// Optional prefix used by `RoomMemberCountIs`
+#[derive(Copy, Clone, Debug, Eq, PartialEq)]
+pub enum RoomMemberCountPrefix {
```

Comment:
Maybe this could have a more generic name, like `CmpOp` (or a longer form). @iinuwa what would you prefer?

Reply:
With just one way to serialize `==X` and `X` the list of options can be reduced to four:

```rust
enum CmpOp {
  Lt,
  Le,
  Gt,
  Ge,
}

struct RoomMemberCountIs {
    prefix: Option<CmpOp>,
    count: UInt,
}
```

Reply:
'Equals' is also a comparison operation, so I think it's fine to keep it as-is.

Reply:
`CmpOp` or `ComparisonOperator` looks good to me.